### PR TITLE
Added a dynamic translation system

### PR DESCRIPTION
**Pull request by yotam180**
Use the default-lang.js file to make more language files.
Added comments in the index.js.

Happy Hacktoberfest.

### CODE REVIEW

**Comment by wolframtheta** [comment]
Oh it's awesome. I want to do this but in another issue... but it seems correct. Thanks @yotam180

**Comment by yotam180** [comment]
Sure. I linked it to that issue 'cause I thought it would be of some help there.
If you want to create a new issue for it then I'll link it to that.
Oh, and feel free to merge the request :wink:

**Comment by wolframtheta** [comment]
Yeah, I will merge later!

**Comment by wolframtheta** [comment]
Can you link to the issue?

https://github.com/wolframtheta/BotCRUPC/issues/4

File: index.js [hunk 1]
```diff
@@ -40,19 +43,19 @@ app.on('sticker', (ctx) => {
...
         console.log("write");
       }
       console.log("closed");
-      ctx.reply('Club cerrado')
+      ctx.reply(dict['7'])
     }
   } 
   else {
-    ctx.reply('A ti no te hago ni puto caso');
+    ctx.reply(dict['8']);
```
Comment (wolframtheta) [inline index.js]: This reply is the same as the number 5

**Review by wolframtheta — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (yotam180) [reply index.js]: Fixed, my friend. Any other things before merging?

**Review by wolframtheta — APPROVED** [review]

**Comment by yotam180** [comment]
Oh, and, #4

**Comment by wolframtheta** [comment]
It's done! Thanks a lot!

**Comment by yotam180** [comment]
Great! You're very welcome :smile:

**Pull request merged**